### PR TITLE
Update patch plugin versions in gemfile lock in preparation for 6.6.2

### PR DESCRIPTION
**Pull request by jsvd**

### CODE REVIEW

**Comment by elasticcla** [comment]
Hi @jsvd, we have found your signature in our records, but it seems like you have signed with a different e-mail than the one used in yout Git [commit](https://github.com/elastic/logstash/pull/10505.patch). Can you please add both of these e-mails into your Github profile (they can be hidden), so we can match your e-mails to your Github profile?

File: Gemfile.jruby-2.3.lock.release [hunk 1]
```diff
@@ -38,27 +38,27 @@ GEM
...
     belzebuth (0.2.3)
       childprocess
     benchmark-ips (2.7.2)
     bindata (2.4.4)
     buftok (0.2.0)
     builder (3.2.3)
     cabin (0.9.0)
-    childprocess (0.9.0)
-      ffi (~> 1.0, >= 1.0.11)
+    childprocess (1.0.1)
```
Comment (jsvd) [inline Gemfile.jruby-2.3.lock.release]:
```suggestion
    childprocess (0.9.0)
```

File: Gemfile.jruby-2.3.lock.release [hunk 2]
```diff
@@ -38,27 +38,27 @@ GEM
...
+      rake (< 13.0)
```
Comment (jsvd) [inline Gemfile.jruby-2.3.lock.release]:
```suggestion
      ffi (~> 1.0, >= 1.0.11)
```

File: Gemfile.jruby-2.3.lock.release [hunk 3]
```diff
@@ -38,27 +38,27 @@ GEM
...
-    aws-sdk-resources (2.11.218)
-      aws-sdk-core (= 2.11.218)
+    aws-sdk-resources (2.11.229)
+      aws-sdk-core (= 2.11.229)
     aws-sdk-v1 (1.67.0)
       json (~> 1.4)
       nokogiri (~> 1)
     aws-sigv4 (1.0.3)
-    backports (3.11.4)
+    backports (3.12.0)
```
Comment (jsvd) [inline Gemfile.jruby-2.3.lock.release]:
```suggestion
    backports (3.11.4)
```

**Pull request merged**